README.md

```diff
@@ -86,3 +86,18 @@ export VSCODE_EXTENSIONS_DIR="$HOME/.vscode-insiders/extensions"
 By default, VS Code changes on the remote server won't be synced back
 when the connection closes. To synchronize back to local when the connection ends,
 pass the `-b` flag.
+
+
+### Auto open in browser trouble shooting
```

Comment:
We can delete this section, I'm going to revert it so that it attempts to open a window everytime, the `DISPLAY` variable won't be consulted.

Reply:
See #90

Reply:
great, it's better for solving from the source.
I'll remove this section.

Reply:
section reverted